code/modules/client/preference_setup/loadout/loadout_eyes.dm

```diff
@@ -147,3 +147,17 @@
 	goggles["goggles, tactical (Zavodskoi)"] = list(/obj/item/clothing/glasses/safety/goggles/goon/zavod, "Zavodskoi Interstellar")
 	goggles["goggles, tactical (Idris)"] = list(/obj/item/clothing/glasses/safety/goggles/goon/idris, "Idris Incorporated")
 	gear_tweaks += new /datum/gear_tweak/path/faction(goggles)
+
+/datum/gear/eyes/medical_goggles
+	display_name = "goggles selection (medical)"
+	description = "A selection of medical goggles. Note that factional ones can only be taken by members of that faction."
+	path = /obj/item/clothing/glasses/safety/goggles/medical
+
+/datum/gear/eyes/medical_goggles/New()
+	allowed_roles = list("First Responder")
```

Comment:
wrong place, goes in `/datum/gear/eyes/medical_goggles`.

Reply:
It still works. I will move it up though.